src/pages/main/hooks/useCmdk.ts

```diff
@@ -242,9 +242,10 @@ export const useCmdk = ({ cmdkReferenceData, importProject }: IUseCmdk) => {
     dispatch(setAutoSave(!autoSave));
   }, [autoSave]);
 
-  const onToggleFormatCode = useCallback(() => {
-    dispatch(setFormatCode(!formatCode));
-  }, [formatCode]);
+  // TODO: it is temorally commented - in future it is possibly used or removed
+  // const onToggleFormatCode = useCallback(() => {
```

Comment:
Remove this even from the redux state as it is going to be on by default.

Reply:
++, removed